daml/Marketplace/Issuance.daml

```diff
@@ -33,6 +42,17 @@ template Service
     maintainer key._1
 
     controller client can
+      nonconsuming RequestOrigination: ContractId OriginationRequest
+        with
+          assetLabel: Text
+          assetSignatories: Set Party
+          description: Text
+          issuer: Party
+          claims: Claims
```

Comment:
As discussed on our call, we should add `RequestOriginationOption`, `RequestOriginationBond` etc. to wrap the creation of `Claims` which are somewhat tricky via the API.
Currently this isn't a problem as we just use it in a script, where we can use the claims combinators to build the instruments, but it will become an issue when tying up with the UI.

Reply:
We should think if those predefined payoffs can live in a difference service, that then uses / calls this choice on here. I could imagine such predefined payoffs to even come in a separate package, which is also how users of the marketplace can extend the functionality.